MiniApp/Classes/JavascriptBridge/MiniAppJavaScriptEnumerations.swift

```diff
@@ -38,6 +40,8 @@ public enum MiniAppCustomPermissionType: String, Codable, CaseIterable {
         return "Profile Photo"
         case .contactsList:
         return "Contact List"
+        case .accessToken:
```

Comment:
Sorry, I forgot to update you about this 😣  We don't want to add a permission for the access token yet. Instead, we will pass the Mini App ID to the Host App and let them verify that the mini app should be able to use the access token.

This is because the access token won't be used by all Host Apps, so we don't want the permission to show up in the permission management screen. In the future, we will change the permissions management functionality so it returns only the permissions which the mini app is actually using (as set in their JSON manifest). Then we can add a permission for access token and anything else that won't be used by all Host Apps.

Reply:
Understood 👍